apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,56 @@
-import { Component, OnInit } from '@angular/core';
+import { NgOptimizedImage } from '@angular/common';
+import { Component, Signal } from '@angular/core';
+import { toSignal } from '@angular/core/rxjs-interop';
+import { switchMap, tap } from 'rxjs';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import {
+  CardComponent,
+  CardListItemDirective,
+} from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  templateUrl: './city-card.component.html',
   standalone: true,
-  imports: [],
+  styles: [
+    `
+      .bg-light-yellow {
+        background-color: rgba(255, 243, 0, 0.1);
+      }
+    `,
+  ],
+  imports: [
+    CardComponent,
+    CardListItemDirective,
+    ListItemComponent,
+    NgOptimizedImage,
+  ],
 })
-export class CityCardComponent implements OnInit {
-  constructor() {}
+export class CityCardComponent {
+  public cities: Signal<Array<City>> = toSignal(
```

Comment:
You could also refactor the `cities$` subject inside `CityStore` to signal and change this to `cities = this.store.cities`.